### PR TITLE
chore: improve release policy

### PR DESCRIPTION
**Pull request by marikaner**
Closes https://github.com/SAP/cloud-sdk-backlog/issues/937.

I removed some information that I found too detailed or duplicate.
I added the announcement link for v3.
Note: I would like to exchange the same file for v2 before merging.

### CODE REVIEW

File: docs-js/release-policy.mdx [hunk 1]
```diff
@@ -19,91 +19,36 @@ keywords:
...
-It's not relevant for the SAP Cloud SDK for Java!
+## Planned Major Release Schedule
 
-:::
+| Version | Status                                                                                                           | Release Date  |
+| :------ | ---------------------------------------------------------------------------------------------------------------- | :------------ |
+| 1.x     | Deprecated                                                                                                       | March 2019    |
+| 2.x     | Released GA                                                                                                      | February 2022 |
+| 3.x     | Planned ([Announcement](https://blogs.sap.com/2022/12/12/the-advent-of-sap-cloud-sdk-for-javascript-version-3/)) | Q1/Q2 2023    |
+| 3.x     | Planned                                                                                                          | Q1/Q2 2024    |
```
Comment (FrankEssenberger) [inline docs-js/release-policy.mdx:31]: [q]I guess this should be 4.x?

**Review by FrankEssenberger — APPROVED** [review]
LGTM only on e question

File: docs-js/release-policy.mdx [hunk 2]
```diff
@@ -19,91 +19,36 @@ keywords:
...
 
-## Planned Major Release Schedule
+### Upgrading to a New Major Version
 
-The SAP Cloud SDK for JavaScript follows [semantic versioning](https://semver.org/).
+- We provide a step-by-step upgrade guide for each major version.
+- Following the upgrade steps should take less than a day of development effort regardless of the project size.
```
Comment (FrankEssenberger) [inline docs-js/release-policy.mdx:36]: [q] Should we link the upgrade guide here?

File: docs-js/release-policy.mdx (hunk 1 above)
Comment (marikaner) [inline docs-js/release-policy.mdx:31]:
```suggestion
| 4.x     | Planned                                                                                                          | Q1/Q2 2024    |
```

**Pull request merged**